GitUp/Application/Base.lproj/MainMenu.xib

```diff
@@ -248,20 +248,47 @@ You must close and reopen any opened repositories in GitUp after changing this s
                                                 </binding>
                                             </connections>
                                         </matrix>
+                                        <popUpButton verticalHuggingPriority="750" id="aYx-qQ-zM6" userLabel="Theme">
+                                            <rect key="frame" x="137" y="34" width="251" height="25"/>
```

Comment:
The popup button is not baseline aligned with the "Theme:" text field. Other than that, it seems to work great!

Reply:
All good!